src/dialog-editor/components/modal-field/field.html

```diff
@@ -42,6 +42,7 @@ <h4 class="modal-title" id="myModalLabel" translate>Edit Field Details</h4>
            pf-label="{{'Dynamic'|translate}}">
         <input bs-switch
                ng-model="vm.modalData.dynamic"
+               ng-change="vm.updateDialogFieldResponders(vm.modalData.name)"
```

Comment:
Should we have a default implementation of `vm.updateDialogFieldResponders`? 

Unconditionally calling an optional function sounds potentially bad...